thirdparty/build-thirdparty.sh

```diff
@@ -836,6 +837,10 @@ build_librdkafka() {
 
     cd "${TP_SOURCE_DIR}/${LIBRDKAFKA_SOURCE}"
 
+    # NOTE(amos): librdkafka uses a weird autoconf variant (mklove) which doesn't allow extending PKG_CONFIG with spaces in cmd.
+    # As a result, we use a patch to hard code "--static" into PKG_CONFIG instead.
+    # PKG_CONFIG="pkg-config --static"
+
```

Comment:
do we need to add --enable-gssapi for build_librdkafka?

Reply:
I don't think so. Linking to sasl2 will pull in all required archives.